src/nodejs/hooks/seeker_agent_test.go

```diff
@@ -0,0 +1,282 @@
+package hooks_test
+
+import (
+	"bytes"
+	"errors"
+	"github.com/cloudfoundry/libbuildpack"
+	"github.com/cloudfoundry/nodejs-buildpack/src/nodejs/hooks"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+var _ = Describe("seekerHook", func() {
+	var (
+		err      error
+		buildDir string
+		depsDir  string
+		depsIdx  string
+		logger   *libbuildpack.Logger
+		stager   *libbuildpack.Stager
+		buffer   *bytes.Buffer
+		seeker   hooks.SeekerAfterCompileHook
+	)
+
+	BeforeEach(func() {
+		buildDir, err = ioutil.TempDir("", "nodejs-buildpack.build.")
+		Expect(err).To(BeNil())
+
+		depsDir, err = ioutil.TempDir("", "nodejs-buildpack.deps.")
+		Expect(err).To(BeNil())
+
+		depsIdx = "07"
+		err = os.MkdirAll(filepath.Join(depsDir, depsIdx), 0755)
+
+		buffer = new(bytes.Buffer)
+		logger = libbuildpack.NewLogger(buffer)
+
+		logger := libbuildpack.NewLogger(os.Stdout)
+		command := MockSeekerCommand{}
```

Comment:
This shadows the above logger. Please remove this so we don't get lots of `stdout` when we run our unit tests

Reply:
removed